.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy
+
+on:
+  push:
+    # TODO: Limit to master branch for trunk deployments and all tags.
+    branches: '**'
+    tags: '**'
+
+jobs:
+
+  deploy:
+    name: Deploy to WordPress.org
+    runs-on: ubuntu-20.04
+    needs: test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          cache: 'npm'
+          node-version-file: '.nvmrc'
+
+      - name: Setup PHP and Composer
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '5.6'
+          tools: composer:v2
+
+      - name: Install NPM dependencies
+        run: npm install
+
+      #- name: Authenticate with WordPress.org SVN
+      #  run: svn co --username ${{ secrets.DEPLOY_SVN_USERNAME }} --password "${{ secrets.DEPLOY_SVN_PASSWORD }}" https://plugins.svn.wordpress.org/two-factor/
```

Comment:
@jeffpaul Do you have suggestions for credentials we could use here? Is there a bot user account that is used for ["beta" plugins](https://wordpress.org/plugins/browse/beta/) like this one?

I see that the performance plugin appears to be using [a set of credentials](https://github.com/WordPress/performance/blob/1410807ccc071c0b00540369f0c1b26debb0032b/.github/workflows/deploy-dotorg.yml#L18-L19) that map to the `performanceteam` user.

Reply:
@kasparsd yes, I can coordinate with folks over in Meta to get something set up to be used.  I'll look to get to that probably post 6.1 Beta next week when I'll have a bit more focus I can direct here.

Reply:
Requested secrets here: https://meta.trac.wordpress.org/ticket/6492

Reply:
@kasparsd SVN_USERNAME + SVN_PASSWORD have been setup, mapping to the `githubsync` WordPress.org user which we use for a few beta plugins with `10up/action-wordpress-plugin-deploy@stable` (Which I would have preferred to see here, but since you have an existing deploy code that already must work.. no objections to using it)